plugin.js

```diff
@@ -25,6 +33,16 @@ function fastifyCookieSetCookie (reply, name, value, options, signer) {
   }
 
   const serialized = cookie.serialize(name, value, opts)
+
+  const bytes = calculateCookieSize(serialized)
```

Comment:
I tried that, but then it didn't match with what my browser (firefox) showed as size

Reply:
So, should this be changed or not?

Reply:
Curious why that `Firefox` doesn't seem to do this - is this still the actual spec? As it would also mean, if a cookie is set on a path like `/a/b/c/d/e/f/g/h/j/k/l/m/n/o/p/q/r/s/t/u/v/w/x/y/z` the cookie size would be mostly used up by it

Reply:
I have no idea what Firefox does. As mentioned a few times, this size limit is not actually a definition in the cookie specification. It is only a consideration. But if we are going to implement it, we should follow the recommendation in the RFC.

Reply:
Most relevant source is RFC 6265 BIS which maintain by httpwg.
https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-rfc6265bis-09#section-5.4

I am not sure if it is a acceptable reference, it seems like it [trace the browser behavior](https://github.com/httpwg/http-extensions/issues/1340) and extend RFC 6265.

Reply:
@climba03003 that is a draft document:

> Internet-Drafts are draft documents valid for a maximum of six months
   and may be updated, replaced, or obsoleted by other documents at any
   time.  It is inappropriate to use Internet-Drafts as reference
   material or to cite them other than as "work in progress."
>
>  This Internet-Draft will expire on 22 April 2022.

I'd rather follow the currently accepted RFC.